src/Components.tsx

```diff
@@ -26,6 +31,7 @@ export const ConnectPaymentDetails = ({
 
   useAttachEvent(paymentDetails, ConnectElementEventNames.close, onClose);
   useAttachAttribute(paymentDetails, 'charge-id', chargeId);
```

Comment:
I'd prefer to remove charge Id altogether - when someone updates they can switch the parameter name. This won't break their old integration.

To me that feels like a minor version bump (not a patch), but not a major version bump.

Reply:
SG, I thought anything non backward-compatible (even if it doesn't break unless someone updates) should be major